src/core/components/Tooltip/index.tsx

```diff
@@ -1,25 +1,96 @@
-import { PropsWithChildren, useId } from 'react';
-import { PlacesType, Tooltip as ReactToolTip } from 'react-tooltip';
+import {
+  arrow,
+  FloatingArrow,
+  offset,
+  useFloating,
+  useHover,
+  useInteractions,
+} from '@floating-ui/react';
+import { useRef, useState } from 'react';
+import clsx from 'clsx';
 
-interface TooltipProps {
-  tooltipContentText: string;
-  placement?: PlacesType;
-  className?: string;
-}
-export default function Tooltip({
-  tooltipContentText,
-  placement = 'top',
-  className = '!bg-primary-03 !rounded-lg',
+import {
+  BUTTON_ROUNDED,
+  ROUNDED,
+} from '@/core/components/Button/ButtonBase/constants';
+import { COLOR_THEME_STYLES } from '@/constants/theme';
+import {
+  ARROW_HEIGHT,
+  FILL_COLOR_THEME_STYLES,
+} from '@/core/components/Tooltip/constants';
+import { TooltipProps } from '@/core/components/Tooltip/types';
+
+const Tooltip = ({
+  content,
   children,
-}: PropsWithChildren<TooltipProps>) {
-  const id = useId();
+  placement,
+  className,
+  rootClassName,
+  hasArrow = true,
+  hidden = false,
+  rounded = ROUNDED['ROUNDED_6'],
+  theme = 'body-02-regular',
+  colorTheme = 'dark',
+  gap = 5,
+}: TooltipProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const arrowRef = useRef(null);
+
+  const {
+    refs: { setFloating, setReference },
+    floatingStyles,
+    context,
+  } = useFloating({
+    open: isOpen,
+    onOpenChange: setIsOpen,
+    placement,
+    middleware: [
+      arrow({
+        element: arrowRef,
+      }),
+      offset(hasArrow ? gap + ARROW_HEIGHT : gap),
+    ],
+  });
+
+  const hover = useHover(context);
```

Comment:
hover에 옵션하나 추가 부탁드립니다!
```suggestion
  const hover = useHover(context, {
    enabled: !disableHover,
  });
```